components/Faq.vue

```diff
@@ -69,33 +36,97 @@ export default {
       required: true
     }
   },
-  computed: {
-    sortedItems() {
-      return [...this.items].sort((a, b) => {
-        if (a.question === 'What is a hackathon?') {
-          return -1
-        } else if (b.question === 'What is a hackathon?') {
-          return 1
-        } else {
-          return 0
-        }
-      })
+  data() {
+    return {
+      leftFaqs: this.getFaqColumn(true),
+      rightFaqs: this.getFaqColumn(false)
     }
   },
   methods: {
-    openSesame: (e) => {
+    toggleBox: (e) => {
       e.target.classList.toggle('active')
       const panel = e.target.nextElementSibling
       if (panel.style.display === 'block') {
         panel.style.display = 'none'
       } else {
         panel.style.display = 'block'
       }
+    },
+    getFaqColumn(left) {
+      const isLeft = (e, i) => left ? i % 2 === 0 : i % 2 !== 0
+      return this.items.filter(isLeft)
+    },
+    // For future usage
+    filterFaq: (category) => {
+      return this.items.filter(i => i.category === category)
+    },
+    prioritizeQuestion: () => {
+      // Have "What is a hackathon?" as the first question
+      this.items.forEach((item, index, object) => {
+        if (item.question === 'What is a hackathon?') {
+          object.splice(index, 1)
+          this.items = [item, ...this.items]
+        }
+      })
+      return this.items
     }
   }
 }
 </script>
 
 <style scoped lang="scss">
-
+@import "bulma/bulma.sass";
+.container {
+  width: 90%;
+  margin: 0 auto;
+}
+.title {
+  text-align: center;
+  font-family: "HK Grotesk Regular";
```

Comment:
Are we specifying HK Grotesk in each component or should we just put it in index?

Reply:
Hmmm I'm not sure. You think we should put it in index?

Reply:
Oh wait the font on figma is Source Sans Pro everywhere so should we be using that?

Reply:
And I think that if its mostly that font, it might be easier to make that the default and put it in index?

Reply:
Yeah you're right, can you approve this now because of the amplify env (hence the bad idea part D:)? I'll apply a font patch soon.

Reply:
Oh yeah np